src/dispatch.jl

```diff
@@ -484,6 +484,29 @@ function Base.:*(α::Number, A::LinearAlgebra.Symmetric{<:AbstractMutable})
     return LinearAlgebra.Symmetric(B, c)
 end
 
+function Base.:*(
+    α::AbstractMutable,
+    A::LinearAlgebra.Symmetric{<:AbstractMutable},
+)
+    c = LinearAlgebra.sym_uplo(A.uplo)
+    B = c == :U ? _mult_upper(α, A) : _mult_lower(α, A)
+    return LinearAlgebra.Symmetric(B, c)
+end
+
+function Base.:*(α::AbstractMutable, A::LinearAlgebra.Symmetric)
+    c = LinearAlgebra.sym_uplo(A.uplo)
+    B = c == :U ? _mult_upper(α, A) : _mult_lower(α, A)
+    return LinearAlgebra.Symmetric(B, c)
```

Comment:
eliminate the `undef`s here as well
```suggestion
    return LinearAlgebra.Symmetric(Matrix(LinearAlgebra.Symmetric(B, c)),c)
end

function Base.:*(α::AbstractMutable, A::LinearAlgebra.Symmetric)
    c = LinearAlgebra.sym_uplo(A.uplo)
    B = c == :U ? _mult_upper(α, A) : _mult_lower(α, A)
    return LinearAlgebra.Symmetric(Matrix(LinearAlgebra.Symmetric(B, c)),c)
```